File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/CountryPresenter.java
package bobrchess.of.by.belaruschess.presenter;

import bobrchess.of.by.belaruschess.view.activity.impl.AuthorizationActivity;

/**
 * Created by Igor on 12.04.2018.
 */

public interface CountryPresenter extends BasePresenter{
    void getCountry(Integer id);

    void attachView(AuthorizationActivity activity);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/CountryPresenter.kt
package bobrchess.of.by.belaruschess.presenter

import bobrchess.of.by.belaruschess.view.activity.CountryPresenterCallBack

/**
 * Created by Igor on 12.04.2018.
 */

interface CountryPresenter : BasePresenter {
    fun getCountry(id: Int?)

    fun getCountries()

    fun attachView(countryPresenterCallBack: CountryPresenterCallBack)
}